loader/data_injector.cpp

```diff
@@ -485,6 +489,8 @@ RetCode DataInjector::LoadFromFileMultiThread(vector<string> input_file_names,
   thread_index_ = 0;
   cout << endl;
 
+  assert(open_flag != FileOpenFlag::kReadFile);
```

Comment:
this assert should be in LoadFromFile() method, where the open_flag parameter is passed in at first